app/javascript/components/input-tags/component.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useRef } from 'react';
+import './styles.scss';
+
+function InputTags() {
+  const [tags, setTags] = useState([]);
+  const inputRef = useRef(null);
+
+  const removeTag = i => {
+    const newTags = [...tags];
+    newTags.splice(i, 1);
+    setTags(newTags);
+  };
+
+  const inputKeyDown = e => {
+    const val = e.target.value;
+
+    if ((e.key === 'Enter' || e.key === ',') && val) {
+      if (e.key === ',' && val) {
+        e.preventDefault();
+      }
+      if (tags.find(tag => tag.toLowerCase() === val.toLowerCase())) {
+        return;
+      }
+      setTags([...tags, val]);
+      // this.tagInput.value = null;
+      if (inputRef.current) {
+        inputRef.current.value = '';
+      }
+    } else if (e.key === 'Backspace' && !val) {
+      removeTag(tags.length - 1);
+    }
+  };
+
+  return (
+    <div className="input-tag">
+      <ul className="input-tag__tags">
```

Comment:
We aren't using this format in other components. Lets follow the same `.c-` structure.